samples/matter/lock/configuration/nrf5340dk_nrf5340_cpuapp/pm_static_dfu.yml

```diff
@@ -45,14 +45,33 @@ mcuboot_secondary_1:
   size: 0x40000
   device: MX25R64
   region: external_flash
+nrf70_wifi_fw_mcuboot_pad:
```

Comment:
Hmm I see we already have `nrf70_wifi_fw` here, so it looks I missed some previous contribution. I don't think it's correct to add these nRF70 related partitions to the nRF53 config file as is. It will be also used by nRF53 Thread configuration, right?

Reply:
Yes, we have had it for some time. There is no problem with having it and using Thread. It only reduces external flash size, but it does not have any impact on the application no matter whether it is Thread or WiFi. We can split those configurations but it will require additional files and code to maintain.

Reply:
I know, but it's similar discussion to the one we had about enabling PSA migration. I'm just generally against adding things that are specific to the one configuration to generic code that in majority of cases does not need it. Such approach is easier to maintain, but it makes basic versions of our apps that will be used in 90% of cases harder to develop .

Imagine being a customer that wants to develop nRF53 Thread door lock. After opening this file I would wonder why the heck I need some nRF70 partition? I don't even know what is nRF70 and I don't have it on my board.

Reply:
Ok, I agree, and as discussed f2f, let's split it.